.github/workflows/pull_request.yml

```
@@ -0,0 +1,21 @@
+name: CI/CD
+on:
+  pull_request:
+    type: [opened, synchronize, closed]
+    branches: [dev]
+
+jobs:
+  test:
+    if: github.event.action == 'opened' || github.event.action == 'synchronize'
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v4
+      - name: Setup node@22
+        uses: actions/setup-node@v3
+        with:
+          node-version: 20
```

```suggestion
         node-version-file: .nvmrc
```

이렇게 하면 .nvmrc 파일 하나로 node version을 관리할 수 있습니다

---

@hayoiii 오오 굳굳 .nvmrc 협업할때 진짜 편하겠군여